terminology/ansi.py

```diff
@@ -183,9 +191,9 @@ def _remove_background_colors(text) -> StyledText:
     return _remove_regex(BACKGROUND_COLORS_REGEX, text)
 
 
-def _remove_bold(text) -> StyledText:
+def _remove_bold_or_dim(text) -> StyledText:
     """Remove all text modifications from the given text."""
```

Comment:
This is unrelated to your change, but this should docstring is 100% incorrect. Are you willing to update it to 

```suggestion
    """Remove bold and dim modifications from the text."""
```